### DISCUSSION TITLE
Chips Theme Variant Inverse

### DISCUSSION THREAD

**Discussion by annefranca** [Telefonica/mistica-design#1774] [Proposals]
### Goal

Avaliar a proposta da implementação do Theme Variant Inverse no componente Chips.

### Context

Necessitamos da aplicação do componente Chips com a Variant Inverse Color em alguns fluxos do Novo App, como em Start Tab e Assinaturas.

### Impact

Ao adicionar o Theme Variant Inverse no componente em questão, estaríamos padronizando uma definição já existente em outros componentes da biblioteca Mística, como:

- Highlighted card
- Snap card
- Header
- Navigation bars
- Boxed row

### Discussion Questions

Vocês têm algum estudo em relação ao Theme Variant Inverse para o componente Chips?

### Do you have a visual proposal?

https://www.figma.com/design/lvpSfY5tWbxfIXDcyOik78/Chips-Theme-Variant-Inverse?node-id=1%3A86&t=igLaeZBwSmM70kVw-1

**Comment by annefranca** [comment]
Link Jira Novum: https://jira.tid.es/browse/OBVIVO-2618

**Comment by HectorSancho** [comment]
Hello!

We need if possible more context (screens or flows).

We also need to know if you really want an inverse over default or alternative (white or grey background).

Or do you want an inverse version that works in over inverse contexts (purple background)

**Reply by annefranca** [reply]
Olá!

Precisamos para aplicação no fundo branco/cinza, segue print de exemplo: 
![Captura de tela 2024-05-21 084622](https://github.com/Telefonica/mistica-design/assets/127137234/14d7996e-9913-4740-99b6-3b0c92c14f84)

**Reply by annefranca** [reply]
@HectorSancho 
Adicionei no arquivo do Figma um caso de uso de aplicação no Novo App.
https://www.figma.com/design/lvpSfY5tWbxfIXDcyOik78/Chips-Theme-Variant-Inverse?node-id=3518%3A17481&t=g28p7Ep2CU4VyQfY-1

**Comment by HectorSancho** [comment]
Hi, we have been discussing this internally this week and we believe that it does not fit the core definition of the component for the following reasons:

- It can be confused with a button
- The user can understand that it is a choice chip and it is selected
- We do not believe it is good practice to mix styles in the same chip group

Anyway, you could do something similar to [this](https://tinyurl.com/2acpfkqc) to highlight the first item

**Comment by piresrodrigo** [comment]
Hola!

Me gustaría darte un poco más de contexto.
Estamos experimentando un problema crítico con los clientes que no encuentran los detalles de su factura. El objetivo es resaltar realmente estos entrypoints de facturas y darles una énfasis de acuerdo con su importancia. Una solución es utilizar también para el chip la función alternativa que ya está disponible para otros componentes en Mistica (como la snapcard). Entiendo que eso seria una evolución que aun estaria alineada con los recursos del Design System. Lo que sería cuestionable es la elección realizada por producto, se seria una buena pratica o no.

Pensamos en optar por seguir como lib extended, pero es un componente que se utiliza en una página CMS. Y por lo tanto, nuestra solicitud de evolución del chip para recibir un color alternativo.

**Reply by yceballost** [reply]
@piresrodrigo tenéis la seguridad de que esto resuelve ese problema? Yo tengo muchas dudas... Cuando un usuario vea una chip "destacada" lo que va a pensar es que ya está en Central de faturas, porque parece que esa chip está selected. 

Creo que se está intentando solucionar un problema de UX y/o arquitectura con Mística y aplicar ese cambio visual no creo que vaya a resolver nada, imho

**Comment by piresrodrigo** [comment]
Entiendo. Tampoco creo que sea la mejor solución, estamos trabajando para que en el Q4 usemos chips con badges de notificación en Facturas. Pero, hasta entonces necesitamos utilizar otros recursos para aliviar el problema crítico de que las facturas no se puedan encontrar. Y por eso la idea fue crear también para el chip la función alternativa que ya está disponible para otros componentes en Mistica.

**Comment by yceballost** [comment]
Esto se incluyó en la release https://github.com/Telefonica/mistica-design/blob/production/changelog-versions/16.4.0.md
